### PR TITLE
Add SUSE-OpenStack-Cloud-7-Pool repo only for upgrade jobs 6_7

### PR DESCRIPTION
**Pull request by gosipyan**

### CODE REVIEW

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -4860,6 +4860,7 @@ function onadmin_prepare_cloudupgrade_admin_repos_6_to_7
 
     # recreate the SUSE-Cloud Repo with the latest iso
     onadmin_prepare_cloud_repos
+    addcloud7pool
```
Comment (jsuchome) [inline scripts/qa_crowbarsetup.sh]: Can you add a comment that we need Pool repos as the `repocheck` step checks for their presence (as they are required for customer setups)?

Comment (gosipyan) [inline scripts/qa_crowbarsetup.sh]: done

**Comment by jsuchome** [comment]
I'd prefer this over the alternative (if it works)

**Comment by gosipyan** [comment]
@jsuchome  testing with https://ci.suse.de/job/openstack-mkcloud/73067/console hope will work .

**Comment by dirkmueller** [comment]
doesn't seem to work :/

**Comment by gosipyan** [comment]
because I did it wrong way updated file manually not giving right PR to jenkins :/ will rebuild

**Comment by gosipyan** [comment]
This should work https://ci.suse.de/job/openstack-mkcloud/73087

**Comment by gosipyan** [comment]
@dirkmueller  @jsuchome  jobs fail but after repocheck so this fixed issue with missing Pool repo  I think.

**Comment by gosipyan** [comment]
I run same command manually to continue nodes upgrade all nodes where upgraded successfully.

**Pull request merged**